packages/react-native-renderer/src/ReactNativeGetListener.js

```diff
@@ -47,7 +47,7 @@ export default function getListener(
   }
 
   // Get imperative event listeners for this event
-  if (stateNode.canonical._eventListeners[registrationName] && stateNode.canonical._eventListeners[registrationName].length > 0) {
+  if (stateNode.canonical?._eventListeners?.[registrationName]?.length > 0) {
```

Comment:
Nit: we generally avoid newer syntax because it's hard to say what Babel will turn it into. (And it's also a bit difficult to remember which Babel plugins we have enabled.)